src/plugins/data_table/filter_view.js

```diff
@@ -68,43 +76,79 @@ var DataTableFilter = (function(options) {
         this.filter.value = filterInput.value;
         this.parentView._toggleIcon(true);
       }
+    },
+
+    afterRender: function() {
+      this.$('.btn-filter').click(this.parentView._onFilterClick.bind(this.parentView));
+      this.$('.btn-clear').click(this.parentView._onClearClick.bind(this.parentView));
```

Comment:
ditto on these afterRender's that are shared with all the filter views.  Esp since they have to call parentView.. seems this stuff should live up on the parent view!